lib/engine.js

```diff
@@ -292,7 +293,11 @@ module.exports = function container (get, set, clear) {
               }
               if (order.status === 'rejected' && (order.reject_reason === 'post only' || api_order.reject_reason === 'post only')) {
                 msg('post-only ' + type + ' failed, re-ordering')
-                return cancelOrder(true)
+                return cb(null, null)
+              }
+              if (order.status === 'rejected' && order.reject_reason === 'balance') {
+                msg('not enough balance for ' + type + ', aborting')
+                return cb(null, null)
```

Comment:
But on the new order execution the balance is adjusted - and that's what we want?